stups-spring-oauth2-server/src/main/java/org/zalando/stups/oauth2/spring/server/DefaultTokenInfoRequestExecutor.java

```diff
@@ -57,55 +62,52 @@ public DefaultTokenInfoRequestExecutor(final String tokenInfoEndpointUrl, RestOp
     }
 
     @Override
-    public Map<String, Object> getMap(String accessToken) {
+    public Map<String, Object> getMap(final String accessToken) {
         return doGetMap(accessToken);
     }
 
     protected Map<String, Object> doGetMap(final String accessToken) {
         logger.debug("Getting token-info from: {}", tokenInfoEndpointUri.toString());
+
         final RequestEntity<Void> entity = buildRequestEntity(tokenInfoEndpointUri, accessToken);
         return restOperations.exchange(entity, TOKENINFO_MAP).getBody();
     }
 
-    //@formatter:off
-    public static RequestEntity<Void> buildRequestEntity(URI tokenInfoEndpointUri, String accessToken) {
-        return RequestEntity.get(tokenInfoEndpointUri)
```

Comment:
A lot of tiny changes that make it really hard to know what's going on.... A unicorn died they said...